.github/workflows/daily-e2e-tests-libvirt.yaml

```diff
@@ -12,135 +12,13 @@ on:
     - cron: '15 4 * * *'
   workflow_dispatch:
 
-env:
-    # cloud-api-adaptor image registry
-    E2E_IMG_REGISTRY: ghcr.io/${{ github.repository_owner }}
-    # cloud-api-adaptor: image release tag
-    E2E_IMG_RELEASE_TAG: latest
-    # cloud-api-adaptor image dev tag
-    E2E_IMG_DEV_TAG: latest-dev
-
 jobs:
-
-  # Build the podvm images.
-  #
-  podvm_builder:
-    uses: ./.github/workflows/podvm_builder.yaml
-    with:
-      registry: ghcr.io/${{ github.repository_owner }}
-      image_tag: latest
-      git_ref: refs/heads/main
-    secrets: inherit
-
-  podvm_binaries:
-    needs: [podvm_builder]
-    uses: ./.github/workflows/podvm_binaries.yaml
-    with:
-      registry: ghcr.io/${{ github.repository_owner }}
-      image_tag: latest
-      git_ref: refs/heads/main
-    secrets: inherit
-
-  podvm:
-    needs: [podvm_binaries]
-    uses: ./.github/workflows/podvm.yaml
+  e2e:
+    uses: ./.github/workflows/e2e_run_all.yaml
     with:
-      registry: ghcr.io/${{ github.repository_owner }}
-      image_tag: latest
+      authorized: true
```

Comment:
IIUC this workflow didn't used the authorized in previous version, is it a mistake or intent?